packages/integrations/cloudflare/src/server.advanced.ts

```diff
@@ -12,6 +12,17 @@ type Env = {
 	name: string;
 };
 
+interface CloudflareContext {
+	locals: {
+		runtime: {
+			waitUntil: (promise: Promise<any>) => void;
+			env: Env;
+			cf: CFRequest['cf'];
+			caches: typeof caches;
+		};
+	};
+}
```

Comment:
this also prepares for a future, where we export an interface for users to extend in their locals type definition.